docs/index.md

```diff
@@ -44,84 +44,46 @@
 
 # Welcome to Streamlit
 
-[Streamlit](https://streamlit.io) is an open-source Python library that makes it
-easy to build beautiful custom web-apps for machine learning and data science.
+[Streamlit](https://streamlit.io/) is an open-source Python library that makes it easy to create and share beautiful, custom web apps for machine learning and data science.
+In just a few minutes you can build and deploy powerful data apps - so let's get started!
 
-To use it, run `pip install streamlit` from a terminal, then import it in Python, write a couple lines
-of code, and run your script with `streamlit run [filename]`. Streamlit watches
-for changes on each save and updates the app live while you're coding. Code
-runs from top to bottom, always from a clean state, and with no need for
-callbacks. It's a simple and powerful app model that lets you build rich UIs
-incredibly quickly. To learn more about how Streamlit works, see [Main
-concepts](main_concepts.md).
+1. Make sure that you have [Python 3.6](https://www.python.org/downloads/) or greater installed.
+2. Install Streamlit using [PIP](https://pip.pypa.io/en/stable/installing/):
 
-You may also want to check out [this four-part
-video](https://www.youtube.com/watch?v=R2nr1uZ8ffc&list=PLgkF0qak9G49QlteBtxUIPapT8TzfPuB8)
-recorded at our PyData talk on December 2019. In it we describe the motivation
-behind Streamlit, then go over how to install and create apps with it.
+   `pip install streamlit`
```

Comment:
For this and all shell commands: Do we want to prefix it with '$'? Like:
```
$ pip install streamlit
```

Pros: it's clear it's supposed to be on the command line
Cons: might make copying a little more annoying

Reply:
We should also break the commands onto their own lines:

````
   ```bash
   streamlit run first_app.py
   ```

   instead of

   `streamlit run first_app.py
````

Reply:
No, `$` as a prefix causes a linter error